src/Parser/Shaman/Restoration/Modules/Spells/HealingRain.js

```diff
@@ -62,6 +62,10 @@ class HealingRain extends Analyzer {
   }
 
   statistic() {
+    if (isNaN(this.averageHitsPerTick)) {
+      return false;
```

Comment:
When you don't want to render anything you should return `null` rather than `false` (I think false triggers a React warning).